src/libraries/Microsoft.PowerFx.Json/FormulaValueJSON.cs

```diff
@@ -17,14 +17,14 @@ public class FormulaValueJSON
         /// <summary>
         /// Convenience method to create a value from a json representation. 
         /// </summary>
-        public static FormulaValue FromJson(string jsonString, FormulaType formulaType = null, bool numberIsFloat = false)
-        {
+        public static FormulaValue FromJson(string jsonString, FormulaType formulaType = null, bool numberIsFloat = false, bool returnUnknownRecordFieldsAsUntypedObjects = false)
```

Comment:
> bool returnUnknownRecordFieldsAsUntypedObjects = false [](http://example.com/codeflow?start=122&length=55)

adding new parameters is a binary breaking  change.

Reply:
I'll check the tooling

Reply:
It wasn't captured as the build failed